touch_element/examples/touch_button/pytest_touch_button.py

```diff
@@ -0,0 +1,15 @@
+# SPDX-FileCopyrightText: 2022-2025 Espressif Systems (Shanghai) CO LTD
+# SPDX-License-Identifier: Unlicense OR CC0-1.0
+import pytest
+from pytest_embedded import Dut
+
+
+@pytest.mark.generic_esp32s2
+@pytest.mark.generic_esp32s3
```

Comment:
the logic looks duplicated. I would still prefer to define the env marker "generic".  WDYT @igrr